### PR TITLE
Fix regression with imported css

### LINKED ISSUE #4986: Polymer 2.3.1: Uncaught TypeError: Cannot read property 'parentNode' of undefined
With Polymer 2.3.1, I get the following error message. Downgrading to Polymer 2.3.0 resolves the issue.

```
Uncaught TypeError: Cannot read property 'parentNode' of undefined
    at processElementStyles (element-mixin.html:431)
    at Function._finalizeTemplate (element-mixin.html:650)
    at HTMLElement._initializeProperties (element-mixin.html:591)
    at HTMLElement._initializeProperties (legacy-element-mixin.html:163)
    at new PropertyAccessors (property-accessors.html:147)
    at new TemplateStamp (template-stamp.html:121)
    at new PropertyEffects (property-effects.html:1134)
    at new PolymerElement (element-mixin.html:559)
    at new GestureEventListeners (gesture-event-listeners.html:46)
    at new Dir (dir-mixin.html:130)
```

### Description
After updating my project to Polymer 2.3.1, the project stopped working with the above error message. In the mentioned location `element-mixin.html:431`, the variable `templateStyle` is undefined, because `templateStyles` is an empty node list.

#### Expected Results
No JS error.

#### Actual Results
JS error.

### Browsers Affected
- [x] Chrome
- [ ] Firefox
- [ ] Edge
- [ ] Safari 9
- [ ] Safari 8
- [ ] IE 11

### Versions
- Polymer: v2.3.1
- webcomponents: v1.0.20

### PR DESCRIPTION
**Pull request by azakus**
Fix case where `<link rel="import" type="css">` is the only styling for
an element

Fixes #4986

### CODE REVIEW

File: lib/mixins/element-mixin.html [hunk 1]
```diff
@@ -255,7 +255,9 @@
       // ensure all gathered styles are actually in this template.
       for (let i=0; i < styles.length; i++) {
         let s = styles[i];
-        let templateStyle = templateStyles[templateStyleIndex];
+        // if no styles in the template, fall back to first element child of template
+        let templateStyle = templateStyles.length ?
+          templateStyles[templateStyleIndex] : template.content.firstElementChild;
```
Comment (TimvdLippe) [inline lib/mixins/element-mixin.html]: Lacking the knowledge, but I assume the `firstElementChild` represents the `style` tag represents the `.css` file?

Comment (sorvell) [reply lib/mixins/element-mixin.html]: This just says put any non-local styles either before the closest local template style or at the top of the template

Comment (azakus) [reply lib/mixins/element-mixin.html]: No, the `<link>` element won't make it into the template.

Comment (sorvell) [reply lib/mixins/element-mixin.html]: This will still throw if there's no `firstElementChild`.

Comment (azakus) [reply lib/mixins/element-mixin.html]: Yup, good point. Let me make the smallest possible repro.

**Pull request merged**